test/black-box/profile/ProfileUpdateCommand.test.ts

```diff
@@ -40,4 +40,19 @@ describe('ProfileUpdateCommand', () => {
         expect(result.address).toBe(generatedProfile.address); // we are not allowing the address to be updated
     });
 
+    test('Should fail to update the Profile due to bad id', async () => {
+        // set up the test data
+        let generatedProfile: any = await testUtil.generateData(CreatableModel.PROFILE, 1, true);
+        generatedProfile = generatedProfile[0];
+        const createdId = -1;
+
+        const profileName = 'UPDATED-DEFAULT-PROFILE-TEST2';
+        const profileAddress = 'UPDATED-DEFAULT-PROFILE-TEST-ADDRESS2';
+        const res = await testUtil.rpc(profileCommand, [profileUpdateCommand, createdId, profileName]);
+
+        res.expectJson();
+        res.expectStatusCode(404);
+        expect(res.error.error.success).toBe(false);
+        expect(res.error.error.message).toBe('Entity with identifier ' + createdId + ' does not exist');
```

Comment:
its better not to test for res.error.error.success to be false, that way you see the real error message immediately.
test the error messages like this, so that we can later modify messages without breaking all the tests: 
expect(res.error.error.message).toBe(new MissingParamException('profileId').getMessage());